.github/workflows/build.yml

```diff
@@ -18,20 +22,23 @@ jobs:
         with:
           fetch-depth: 0  # fetches all commits/tags
 
-      - name: Set dynamic variables
-        # this might seem odd but is the recommended way to set variables
-        # reference: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: |
-          pip install poetry
-          BUILD_VERSION=$(poetry version -s)
-          echo "BUILD_VERSION=$BUILD_VERSION+$GITHUB_SHA" >> $GITHUB_ENV
-          echo "IMAGE_NAME_WITH_TAG=quipucords:$BUILD_VERSION" >> $GITHUB_ENV
-
-      - name: Docker Layer Caching
```

Comment:
Since we move to podman / buildah, docker layer caching is no longer useful for us. If image building performance becomes a bottleneck, we might need to think about bringing this back.